src/routes/about/+page.svelte

```diff
@@ -1,12 +1,30 @@
 <script>
     import Breadcrumb from "$lib/components/Breadcrumb.svelte";
     import ToolTip from "$lib/components/ToolTip.svelte";
+    import Metatag from '$lib/components/Metatag.svelte';
     const flow = [
         {name: "Home", link: "/"},
         {name: "About", link: "/about"},
     ];
 </script>
 
+<Metatag data={{
+	title:"About Page",
+	description:"SCIS WEB About Page",
+	link:'http://localhost:5173/about',
```

Comment:
For all the places where we have the `http://<domain>:<port>`, add env vars

like `SITE_URL` can be one variable for the actual page domain
`STRAPI_UPLOAD_URL` can be another variable for the strapi's upload path.

also the thing is, we wont be using the `strapi`'s endpoint in production. we will either use the `SITE_URL` (because nginx can handle the routing) or just use `/uploads/...` instead of adding the domain